TAF_EasyRest/TestFramework/TestFramework.csproj

```diff
@@ -20,8 +20,4 @@
     <PackageReference Include="Selenium.WebDriver.ChromeDriver" Version="103.0.5060.5300" />
   </ItemGroup>
 
-  <ItemGroup>
-    <Folder Include="PageComponents\Moderator\" />
```

Comment:
Why it has been removed?

Reply:
It was an empty folder. Now there are some files there and the folder still exists.